src/api/partitioned-dbs.rst

```diff
@@ -0,0 +1,198 @@
+    and active data sizes.
+
+    :code 200: Request completed successfully
+
+    **Request**:
+
+    .. code-block:: http
+
+        GET /db/_partition/foo HTTP/1.1
+        Accept: application/json
+        Host: localhost:5984
+
+    **Response**:
+
+    .. code-block:: http
+
+        HTTP/1.1 200 OK
+        Cache-Control: must-revalidate
+        Content-Length: 119
+        Content-Type: application/json
+        Date: Thu, 24 Jan 2019 17:19:59 GMT
+        Server: CouchDB/2.3.0-a1e11cea9 (Erlang OTP/21)
+
+        {
+          "db_name": "my_new_db",
+          "doc_count": 1,
+          "doc_del_count": 0,
+          "partition": "sensor-260",
+          "sizes": {
+            "active": 244,
+            "external": 347
+          }
+        }
+
+
+``/db/_partition/partition/_all_docs``
+======================================
+
+.. http:get:: /{db}/_partition/{partition}/_all_docs
+    :synopsis: Return all docs in the specified partition
+
+    :param db: Database name
+    :param partition: Partition name
+
+    This endpoint is a convenience endpoint for automatically setting
+    bounds on the provided partition range. Similar results can be had
+    by using the global ``/db/_all_docs`` end point with appropriately
+    configured values for ``start_key`` and ``end_key``.
+
+    Refer to the :ref:`view endpoint <api/ddoc/view>` documentation for
+    a complete description of the available query parameters and the format
+    of the returned data.
+
+    **Request**:
+
+    .. code-block:: http
+
+        GET /db/_partition/sensor-260/_all_docs HTTP/1.1
+        Accept: application/json
+        Host: localhost:5984
+
+    **Response**:
+
+    .. code-block:: http
+
+        HTTP/1.1 200 OK
+        Cache-Control: must-revalidate
+        Content-Type: application/json
+        Date: Sat, 10 Aug 2013 16:22:56 GMT
+        ETag: "1W2DJUZFZSZD9K78UFA3GZWB4"
+        Server: CouchDB (Erlang/OTP)
+        Transfer-Encoding: chunked
+
+        {
+          "offset": 0,
+          "rows": [
+            {
+              "id": "sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+              "key": "sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+              "value": {
+                "rev": "1-05ed6f7abf84250e213fcb847387f6f5"
+              }
+            }
+          ],
+          "total_rows": 1
+        }
+
+
+
+``/db/_partition/partition/_design/design-doc/_view/view-name``
+===============================================================
+
+.. http:get:: /{db}/_partition/{partition}/_design/{ddoc}/_view/{view}
+    :synopsis: Execute a partitioned query
+
+    :param db: Database name
+    :param partition: Partition name
+    :param ddoc: Design document id
+    :param view: View name
+
+    This endpoint is responsible for executing a partitioned query. The
+    returned view result will only contain rows related to the specified
```

Comment:
```suggestion
    returned view result will only contain rows stored on the specified partition.
```

Is this accurate? If so, it seems more precise.

Reply:
Re-reading today I went with "only contain rows with the specified partition name". "stored on" seems not quite right and I agree that "related to" seems a bit of a weak association there.

Reply:
Addressed in 347a6ce.